stable/prometheus-redis-exporter/values.yaml

```diff
@@ -16,6 +16,13 @@ image:
   tag: v0.16.0
   pullPolicy: IfNotPresent
 extraArgs: {}
+# Additional Environment variables
+env:
+  # - name: REDIS_PASSWORD
```

Comment:
Should it be initialized to empty map by default?
Better update Readme too for documentation